docs/calendar.md

```diff
@@ -41,6 +41,8 @@ For specialized applications where you need to do calculations in a calendar sys
 To do this, create a class inheriting from `Temporal.Calendar`, call `super()` in the constructor with a calendar identifier, and implement all the methods.
 Any subclass of `Temporal.Calendar` will be accepted in Temporal APIs where a built-in `Temporal.Calendar` would work.
 
+The identifier of a custom calendar must not be any of the built-in calendar identifiers, and must consist of one or more components of between 2 and 8 ASCII alphanumeric characters each, separated by dashes, as described in [BCP 47](https://tools.ietf.org/html/bcp47#section-2.1).
```

Comment:
Why is it explicitly illegal for a custom calendar to have the same ID as a built-in calendar?

Reply:
Now that I think about it, I guess we do want to allow this, for overriding the built-in ones.